src/Projekt/Main.fs

```diff
@@ -24,10 +25,17 @@ let main argv =
         match result with
         | Success el -> save el path
         | Failure msg -> eprintfn "%s" msg; 1
+
+    let templatesDir = 
+        //slight hack to allow the templates directory to be located 
+        //in the same directory as the exe for distribution 
+        let cur = Reflection.Assembly.GetExecutingAssembly().Location </> "templates"
```

Comment:
This suggests we should use something slightly different due to shadow-copying: http://blog.adilakhter.com/2007/12/17/get-current-executing-assemblys-directory/